frontend/src/renderer/components/ImageImporter.svelte

```diff
@@ -0,0 +1,148 @@
+<script lang="ts">
+    import { FilePlusIcon } from "svelte-feather-icons";
+    import { processState, sendMessage, messageStore } from "@util/stores";
+    import { forEach, find } from "lodash";
+    import ImageBubble from "@components/Process/ImageBubble.svelte";
+    import Dropzone from "svelte-file-dropzone";
+    export let label = "Select Files";
+    export let icon = FilePlusIcon;
+    export let largeText = false;
+    let filePaths = [];
+    let files = {
+        accepted: [],
+        rejected: []
+    };
+    let MAX_FILES = 8
+
+    function getThumbnails() {
+        console.log("Getting Thumbnails");
+        $processState.thumbnailID = Math.floor(Math.random() * 999999999);
```

Comment:
Not a big deal but I'm seeing a typescript error on the above line because the guy who originally wrote this gave it type 'null'. You can fix it by fixing the line where `$processState.thumbnailID` is declared in `stores.ts` to this: `thumbnailID: null as number,`